app/build.gradle

```diff
@@ -19,7 +19,7 @@ android {
     }
   }
   dexOptions {
-    preDexLibraries !"true".equals(System.getenv("TRAVIS"))
+    preDexLibraries "true" != System.getenv("TRAVIS")
```

Comment:
Just curious, do we need if that's still needed?

Reply:
You're right, it's not needed anymore. Removed.